src/main/pages/che-7/administration-guide/proc_configuring-prod-short-to-use-external-keycloak.adoc

```diff
@@ -3,44 +3,44 @@
 [id="configuring-prod-short-to-use-external-keycloak_{context}"]
 = Configuring {prod-short} to use an external {identity-provider} installation
 
-By default, {prod-short} installation in multiuser mode includes the deployment of a dedicated {identity-provider} instance. But using an external {identity-provider} is also possible.
+By default, {prod-short} installation in multiuser mode includes the deployment of a dedicated {identity-provider} instance. However, using an external {identity-provider} is also possible. This option is useful when a user has an already existing {identity-provider} instance with already defined users.
 
 .Prerequisites
 
 ifeval::["{project-context}" == "che"]
-
-* This procedure is only applicable to {prod-short} installations done with the {prod-short} Operator. When installing with the `{prod-cli}` management tool and the `helm` installation mode, there is no easy way to use an external {identity-provider} instance.  
-
+* This procedure is only applicable to {prod-short} installations done using the {prod-short} Operator. When using the `{prod-cli}` management tool and the `helm` installation mode to install {prod-short}, no supported method is available to use an external {identity-provider} instance.  
 endif::[]
 
-*  In the administration console of the {identity-provider} external installation, you shoud have defined a link:https://www.keycloak.org/docs/latest/server_admin/#_create-realm[realm] that will contain the users who should connect to {prod-short}. Let's call it `realm-for-users` in the following, but in the next steps you would replace this name by the real one.
-
+*  In the administration console of the {identity-provider} external installation, define a link:https://www.keycloak.org/docs/latest/server_admin/#_create-realm[realm] that will contain the users intended to connect to {prod-short}. The following steps use the example `realm-for-users` realm name.
```

Comment:
In fact maybe this realm already exist and has already be defined (for example in a company-wide Keycoak instance), but can be reused for Che.

So we should either:
- identify an already-existing that contains the users we're interested in
- Or create one if there's none.

Reply:
Any steps that could tell that the realm already exists and could be used for Che @davidfestal ?